tests/test_send_dos_opportunities_email.py

```diff
@@ -155,7 +166,11 @@ def test_get_campaign_data():
 def test_main_creates_campaign_sets_content_and_sends_campaign(
     get_campaign_data, get_live_briefs_between_two_dates, get_html_content
 ):
-    get_live_briefs_between_two_dates.return_value = [{"brief": "yaytest"}]
+    live_briefs = [
+        {"brief": "yaytest", "frameworkName": "Digital Outcomes and Specialists 2"},
+        {"brief": "bootest", "frameworkName": "Digital Outcomes and Specialists 1"},
```

Comment:
Would `Digital Outcomes and Specialists 50` give the right suffix here? I'm not sure we really need to support it - DOS 10, can you imagine 🙀 - but having the `50` test scenario above seems a bit misleading so I'd ditch that.

Reply:
I think I was just trying to show that it's not limited to just "1" and "2". Happy to remove though.

Reply:
It's gone!